tests/parser.cc

```diff
@@ -417,7 +417,7 @@ TEST_CASE("correct use of Mata::Parser::parse_mf_section()")
 		REQUIRE(ref->size() == 0);
 
 		// check what remains
-		std::string remains = stream.str().substr(stream.tellg());
+		std::string remains = stream.str().substr(static_cast<unsigned long>(stream.tellg()));
```

Comment:
This should be `size_t`. As explained above. I will make all the changes I suggested here and we can immediately merge the PR.

Reply:
Done.